lib/cocoapods/generate/installer.rb

```diff
@@ -64,7 +64,12 @@ def install!
           end
 
           UI.section 'Performing post-installation steps' do
-            perform_post_install_steps(open_app_project, installer)
+            should_perform_post_install = if installer.respond_to?(:generated_aggregate_targets) # CocoaPods 1.7.0
```

Comment:
CocoaPods 1.7.0 can tell us whether we need to perform post installation if the aggregate target was integrated.